gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorage.java

```diff
@@ -439,6 +424,9 @@ GoogleCloudStorageItemInfo composeObjects(
       List<StorageResourceId> sources, StorageResourceId destination, CreateObjectOptions options)
       throws IOException;
 
+  /** Retrieves the statistics associated with the {@link GoogleCloudStorage} instance. */
+  Map<String, Long> getStatistics();
```

Comment:
nit : do we want to add a default impl ?

Reply:
Ack, will follow up on this.